addOns/selenium/src/main/zapHomeFiles/scripts/templates/selenium/Selenium default template.js

```diff
@@ -0,0 +1,13 @@
+// The browserLaunched function is called whenever a browser is launched from ZAP using selenium.
```

Comment:
Worth adding JSDoc describing the parameters.

Reply:
Have added comments to the default template.